invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/ItemsSearch.js

```diff
@@ -44,6 +47,24 @@ const resultsPerPageValues = [
   },
 ];
 
+const config = invenioConfig.items.search;
```

Comment:
Split this part to a `config.js` file?

Reply:
I would say to expose them under a const config variable, so here we can reference them as `config.xxx`, just to know immediately from where they are coming.

Reply:
Done!